openmetadata-ui/src/main/resources/ui/src/utils/GlobalSettingsUtils.tsx

```diff
@@ -227,55 +218,37 @@ export const getGlobalSettingsMenuWithPermission = (
       items: [
         {
           label: i18next.t('label.table-plural'),
-          isProtected: userPermissions.hasViewPermissions(
-            ResourceEntity.TYPE,
-            permissions
-          ),
+          isProtected: Boolean(isAdminUser),
           key: 'customAttributes.tables',
           icon: <TableIcon className="side-panel-icons" />,
         },
         {
           label: i18next.t('label.topic-plural'),
-          isProtected: userPermissions.hasViewPermissions(
-            ResourceEntity.TYPE,
-            permissions
-          ),
+          isProtected: Boolean(isAdminUser),
           key: 'customAttributes.topics',
           icon: <TopicIcon className="side-panel-icons" />,
         },
         {
           label: i18next.t('label.dashboard-plural'),
-          isProtected: userPermissions.hasViewPermissions(
-            ResourceEntity.TYPE,
-            permissions
-          ),
+          isProtected: Boolean(isAdminUser),
           key: 'customAttributes.dashboards',
           icon: <DashboardIcon className="side-panel-icons" />,
         },
         {
           label: i18next.t('label.pipeline-plural'),
-          isProtected: userPermissions.hasViewPermissions(
-            ResourceEntity.TYPE,
-            permissions
-          ),
+          isProtected: Boolean(isAdminUser),
```

Comment:
@harshach just wanted to confirm this change,
Admin won't be able to allow access to these pages unless they make User an Admin
Earlier it was Roles / Policy-based

Reply:
only subsection right
i.e 
<img width="527" alt="image" src="https://github.com/open-metadata/OpenMetadata/assets/38649/5675d123-8e64-4812-a26e-21dd7f022a0a">
Users shouldn't be able to see them unless they are admin

Reply:
Okay got it